sql-plugin/src/main/java/com/nvidia/spark/rapids/ColumnarCopyHelper.java

```diff
@@ -162,7 +167,33 @@ public static void stringCopy(ColumnVector cv, ColumnBuilder b, int rows) {
     }
   }
 
-  // TODO: https://github.com/NVIDIA/spark-rapids/issues/4784
+  public static void decimal32CopyFromParquet(WritableColumnVector cv,
```

Comment:
This method should not have`Parquet` in the method name.  `WritableColumnVector` may be used by Spark's Parquet code, but it's not necessarily specific to Parquet (it's also used by `InMemoryRelation` in some cases).  I'd just call this `decimal32Copy` and let the type overloading handle cases where we want to copy from different column types.

Comment applies to the other decimal copy methods that were added.

Reply:
Applied.